### PR TITLE
Update dependencies.md

### PR DESCRIPTION
**Pull request by piotrpio**
Signed-off-by: Piotr Piotrowski <piotr@synadia.com>

### CODE REVIEW

File: dependencies.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
-| github.com/nats-io/nats.go                       | Apache License 2.0                      |
-| golang.org/x/net                                 | BSD 3-Clause "New" or "Revised" License |
-| golang.org/x/text                                | BSD 3-Clause "New" or "Revised" License |
-| gopkg.in/yaml.v3                                 | Apache License 2.0                      |
+| Dependency                                | License      |
+|-------------------------------------------|--------------|
+| github.com/antonmedv/expr v1.15.3         | MIT          |
+| github.com/dustin/go-humanize v1.0.1      | MIT          |
+| github.com/google/go-cmp/cmp v0.5.9       | BSD-3-Clause |
+| github.com/klauspost/compress v1.17.0     | Apache-2.0   |
```
Comment (ripienaar) [inline dependencies.md:10]: Do we really need versions here? This seems like a recipe for immediatley being out of date as we keep updating dependencies.

Also dont seem worth listing like `compress` and `compress/v2`?

Comment (ripienaar) [reply dependencies.md:10]: Here's an example from a previous cycle PR https://github.com/nats-io/natscli/pull/560/files

No versions, this seems extremely onerous if not strictly needed

Comment (piotrpio) [reply dependencies.md:10]: You're right, versions may be a bit much, I'll remove them.

The thing with `compress` is that `compress/s2` actually has a different license file: https://github.com/klauspost/compress/blob/master/s2/LICENSE

Comment (ripienaar) [reply dependencies.md:10]: ah nice, missed the seperate license, good catch

Comment (piotrpio) [reply dependencies.md]: I removed the versions

File: dependencies.md [hunk 2]
```diff
@@ -2,14 +2,16 @@
...
-| github.com/nats-io/nats.go                       | Apache License 2.0                      |
-| golang.org/x/net                                 | BSD 3-Clause "New" or "Revised" License |
-| golang.org/x/text                                | BSD 3-Clause "New" or "Revised" License |
-| gopkg.in/yaml.v3                                 | Apache License 2.0                      |
+| Dependency                        | License      |
+|-----------------------------------|--------------|
+| github.com/antonmedv/expr         | MIT          |
+| github.com/dustin/go-humanize     | MIT          |
+| github.com/google/go-cmp/cmp      | BSD-3-Clause |
+| github.com/klauspost/compress     | Apache-2.0   |
```
Comment (ripienaar) [inline dependencies.md:10]: Actually this is wrong, looking at the repo he has a bunch of licences in the one file and the MIT one is first and applies to the main repo, I think it was right before @piotrpio

Comment (ripienaar) [reply dependencies.md:10]: Actually, I am not even sure what that license is, but its not ASL-2 or MIT

Comment (ripienaar) [reply dependencies.md:10]: It's `BSD-3-Clause` I guess, same as go

**Pull request merged**